mscore/plugin/api/score.h

```diff
@@ -73,36 +88,48 @@ class Score : public Ms::PluginAPI::ScoreElement {
 
       Ms::Score* score() { return toScore(e); }
       const Ms::Score* score() const { return toScore(e); }
+
+      QString composer() { return score()->metaTag("composer"); }
+      int duration() { return score()->duration(); }
+      int harmonyCount() { return score()->harmonyCount(); }
+      bool hasHarmonies() { return score()->hasHarmonies(); }
+      bool hasLyrics() { return score()->hasLyrics(); }
+      int keysig() { return score()->keysig(); }
+      int lyricCount() { return score()->lyricCount(); }
+      QString poet() { return score()->metaTag("lyricist"); } // not the meanwhile obsolete "poet"
```

Comment:
maybe better name it `lyricist`? And/or return ` metaTag("poet")` if `metaTag("lyricist")` is empty?

Reply:
By the way, the implementation of these properties (poet, composer etc.) in MuseScore 2 tried to search for the text with the relevant style in a score itself and got a value from a meta tag only if that text is not found (see the old [score.cpp](https://github.com/musescore/MuseScore/blob/2.3.2/libmscore/score.cpp#L3859)). Could this perhaps make sense in the new API too?

Reply:
IIRC MuseScore 2 called it poet in plugins, but picked on the meta tag lyricist, so MuseScore 3 should do too I guess

Reply:
Yes, but should we also apply the logic of searching for the text inside score too?

Reply:
How? By copying that old code?

Reply:
Well, it looks like there are no such functions in libmscore now. There is [a similar code](https://github.com/musescore/MuseScore/blob/13a720151231ef749e64107706652260ecf62660/mscore/file.cpp#L3080) in `saveMetadataJSON`, we can try reusing it (by extracting to separate functions again or using a simplified version returning something like `score->getText(Tid::POET)->plainText()` only).
Or we can postpone this to the future revisions, perhaps then the documentation should be corrected not to state that we intend to use metatags only.

In any case, let's better name it `lyricist` indeed, returning a `lyricist` tag contents on calling `poet()` may be somewhat confusing.

Reply:
changed to lyricist, leaving the reste for later...